mainTemplate.json

```diff
@@ -150,7 +142,7 @@
             "minLength": 1,
             "defaultValue": "https://raw.githubusercontent.com/marklogic/cloud-enablement-azure/v9.0-6.1"
         },
-        "location": {
+        "Location": {
```

Comment:
Is this change necessary? Have you tried if it works with the createUiDefinition?

Reply:
The `Location` parameter in the createUiDefinition.json does have a capital *L*. I change this for this reason.